decidim-admin/app/commands/decidim/admin/destroy_feature.rb

```diff
@@ -14,7 +14,11 @@ def initialize(feature)
       #
       # Broadcasts :ok if it got destroyed, raises an exception otherwise.
       def call
-        destroy_feature
+        begin
+          destroy_feature
+        rescue StandardError
```

Comment:
I get what you're doing here, but I feel like rescuing `StandardError` can be problematic, specially if we aren't capturing theses exceptions - they would fail silently in production.

Shouldn't we be appending the error message to feature model, even if it isn't useful to the user?